apps/api-status/development.yml

```
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: blackbox-exporter
+  namespace: argocd
+spec:
+  project: default
```

```suggestion
  project: prod
```
We use the prod project when pointing at prod.